voltixApp/VoltixApp/Stores/UserPreferencesStore.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  UserDefaults.swift
+//  VoltixApp
+//
+//  Created by Enrique Souza Soares on 11/04/2024.
+//
+
+import Foundation
+
+class UserPreferencesStore {
+    static var currency: String {
```

Comment:
Does it more convenient to return SettingsCurrency from store?

Reply:
What do you mean? To rename it?

Reply:
I got it, makes sense.